cmd/osctl/pkg/client/config/config.go

```diff
@@ -18,12 +18,26 @@ type Config struct {
 	Contexts map[string]*Context `yaml:"contexts"`
 }
 
+func (c *Config) upgrade() {
+	for _, ctx := range c.Contexts {
+		ctx.upgrade()
+	}
+}
+
 // Context represents the set of credentials required to talk to a target.
 type Context struct {
-	Target string `yaml:"target"`
-	CA     string `yaml:"ca"`
-	Crt    string `yaml:"crt"`
-	Key    string `yaml:"key"`
+	DeprecatedTarget string   `yaml:"target,omitempty"` // Field deprecated in favor of Endpoints
+	Endpoints        []string `yaml:"endpoints"`
```

Comment:
How difficult would it be to add a `Nodes []string` field here? It would be awesome if we didn't have to type out which nodes we want to target every time.

Reply:
default to empty? and command to set them?

Reply:
added, acts as default value for `--nodes`